libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-adapter.service.ts

```diff
@@ -1,6 +1,9 @@
 import { Injectable } from '@angular/core';
 import { SkyCoreAdapterService } from '@skyux/core';
 
+/**
+ * @internal
```

Comment:
This file isn't exported, but I added the `@internal` just so we know at a glance that this service isn't part of the public API.